modules/board.py

```diff
@@ -17,21 +18,30 @@ def __init__(self, player_colors, board=None):
         if board is None:
             self._board = [[None for _ in range(8)] for _ in range(8)]
```

Comment:
Det lader til, at jeg er kommet til at skrive det samme to gange. Det er vist sådan noget der sker når man ikke har review på! xD Du må meget gerne bare slette den først gang boardet er defineret og kun bruge den anden. Den anden er en nyere version hvor jeg bruger funktionen ```__setitem__()``` til at skrive det pænere. :)